.github/release.yml

```diff
@@ -6,13 +6,60 @@ changelog:
     labels:
       - passed first triage
       - passed secondary triage
+      - team-release
+      - "team: flakes"
+      - revert
   categories:
     - title: Framework
       labels:
         - framework
-    - title: Tooling
+        - "a: text input"
+        - "f: scrolling"
+        - "f: routes"
+        - "f: selection"
+      exclude:
+        labels:
+          - "f: material"
+    - title: Material
       labels:
-        - tool
+        - "f: material"
+    # Mobile
+    - title: iOS
+      labels:
+        - platform-ios
+        - "f: cupertino"
+    - title: Android
+      labels:
+        - platform-android
+    # Desktop
     - title: MacOS
       labels:
         - platform-mac
+    - title: Windows
+      labels:
+        - platform-windows
+    - title: Linux
+      labels:
+        - platform-linux
```

Comment:
Is it worth adding a general/other "Desktop" category that has the `"a: desktop"` label but excludes ones with any of the platform specific ones added? I see a non-insignificant amount of PRs that match that situation.

Reply:
This makes sense to have a catch-all desktop category for things that are cross desktop @cbracken does this occur enough to be something we should consider?